scripts/lint_prowjobs/main.go

```diff
@@ -0,0 +1,290 @@
+		}
+		return true, "", ""
+	})
+}
+
+func PresubmitMakeTargetCheck(jc *JobConstants) presubmitCheck {
+	return presubmitCheck(func(presubmitConfig config.Presubmit, fileContentsString string) (bool, string, string) {
+		jobMakeTargetMatches := regexp.MustCompile(`make (\w+) .*`).FindStringSubmatch(strings.Join(presubmitConfig.JobBase.Spec.Containers[0].Command, " "))
+		jobMakeTarget := jobMakeTargetMatches[len(jobMakeTargetMatches)-1]
+		makeCommandLineNo := findLineNumber(fileContentsString, "make")
+		if strings.Contains(presubmitConfig.JobBase.Name, "cni") && jobMakeTarget != jc.CniMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.CniMakeTarget)
+		} else if strings.Contains(presubmitConfig.JobBase.Name, "helm-chart") && jobMakeTarget != jc.HelmMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.HelmMakeTarget)
+		} else if strings.Contains(presubmitConfig.JobBase.Name, "release-tooling") && jobMakeTarget != jc.ReleaseToolingMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.ReleaseToolingMakeTarget)
+		} else if jobMakeTarget != jc.DefaultMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.DefaultMakeTarget)
+		}
+		return true, "", ""
+	})
+}
+
+func PostsubmitMakeTargetCheck(jc *JobConstants) postsubmitCheck {
+	return postsubmitCheck(func(postsubmitConfig config.Postsubmit, fileContentsString string) (bool, string, string) {
+		jobMakeTargetMatches := regexp.MustCompile(`make (\w+) .*`).FindStringSubmatch(strings.Join(postsubmitConfig.JobBase.Spec.Containers[0].Command, " "))
+		jobMakeTarget := jobMakeTargetMatches[len(jobMakeTargetMatches)-1]
+		makeCommandLineNo := findLineNumber(fileContentsString, "make")
+		if jobMakeTarget != jc.DefaultMakeTarget {
+			return false, makeCommandLineNo, fmt.Sprintf("Invalid make target, please use the \"%s\" target", jc.DefaultMakeTarget)
+		}
+		return true, "", ""
+	})
+}
+
+func getFilesChanged(gitRoot string, pullBaseSha string, pullPullSha string) ([]string, []string, error) {
+	presubmitFiles := []string{}
+	postsubmitFiles := []string{}
+	gitDiffCommand := "git -C " + gitRoot + " diff --name-only " + pullBaseSha + " " + pullPullSha
+	fmt.Println(gitDiffCommand)
+
+	gitDiffOutput, err := exec.Command("git", strings.Split(gitDiffCommand, " ")[1:]...).Output()
+	filesChanged := strings.Fields(string(gitDiffOutput))
+	for _, file := range filesChanged {
+		if strings.Contains(file, "presubmits") {
+			presubmitFiles = append(presubmitFiles, file)
+		}
+		if strings.Contains(file, "postsubmits") {
+			postsubmitFiles = append(postsubmitFiles, file)
+		}
+	}
+	return presubmitFiles, postsubmitFiles, err
+}
+
+func unmarshalJobFile(filePath string, jobConfig *config.JobConfig) (string, string, string, *config.JobConfig) {
```

Comment:
Can you give the 3 string response parameters names for clarity or add a doc string? Alternatively, it might be helpful to have a type since you have 3 parameters associated with a `config.JobConfig` and may want more

```go
type JobFile struct {
  jobConfig *config.JobConfig
  filePath string
  fileContent string
  GitHubOrg string
}
```